detox/android/rninfo.gradle

```diff
@@ -1,11 +1,24 @@
 import groovy.json.JsonSlurper
 
-def getRNVersion = { workingDir ->
-    println("RNInfo: workingDir=$workingDir")
+def getRNPackageJsonDir() {
+    def currentDir = rootDir
+    while (currentDir != null) {
+        def nodeModulesDir = new File(currentDir, "node_modules/react-native")
+        def file = new File(nodeModulesDir, 'package.json')
+        if (file.exists()) {
+            return file.path
+        }
+        currentDir = currentDir.parentFile
```

Comment:
Will this continue all the way up to the filesystem's root? I think we should stop at the root project's dir 🤔

Reply:
We don't know who is the root. It could be in any structure of projects in mono repo. It is logn. not so bad

Reply:
Yeah I didn't have performance in mind when I wrote this, rather possible resolutions of wrong node module folders or attempting access to non-priviledged directories (think CI agents).
Anyways, smt to think abt but might be an overkill.